proto/message_api/v1/message_api.proto

```diff
@@ -7,50 +7,54 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/xmtp/proto/v3/go/message_api/v1";
 option java_package = "org.xmtp.proto.message.api.v1";
-
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
-    info: {
-        title: "MessageApi";
-        version: "1.0";
-    };
-};
+    info: { title: "MessageApi";
+version: "1.0";
+}
+;
+}
+;
```

Comment:
Is this what the formatter is forcing? This looks terrible.

Reply:
Completely agree. I tried using a different formatter and that one fails the linter. So, I'm kinda stuck.